### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by marcoscaceres**
Dependencies were getting a little old.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,23 +4,23 @@
...
-    "cors": "2.4.2",
-    "errorhandler": "1.2.0",
-    "express": "4.9.5",
-    "multer": "0.1.4"
+    "body-parser": "1.15.2",
+    "compression": "1.6.2",
+    "cors": "2.8.1",
+    "errorhandler": "1.5.0",
+    "express": "4.14.0",
+    "multer": "^1.2.1"
```
Comment (tobie) [inline package.json]: Do we still need this dep?

File: package.json [hunk 2]
```diff
@@ -4,23 +4,23 @@
...
+    "async": "2.1.4",
+    "ent": "2.2.0",
+    "jsdom": "9.9.1",
+    "json-pointer": "0.6.0",
+    "mocha": "3.2.0",
+    "request": "2.79.0",
+    "tv4": "1.2.7",
+    "tv4-formats": "2.2.1",
+    "xml2js": "0.4.17"
   },
```
Comment (tobie) [inline package.json]: Have you checked running the update scripts with those updates?

File: package.json (hunk 1 above)
Comment (marcoscaceres) [inline package.json]: Dunno, it was in the original set. Maybe we don't actually need it?

File: package.json (hunk 2 above)
Comment (marcoscaceres) [inline package.json]: I only did "npm test".

File: package.json (hunk 1 above)
Comment (tobie) [inline package.json]: Well, you removed the line that requires it, so…

File: package.json (hunk 2 above)
Comment (tobie) [inline package.json]: Oh, boy, that test suite is a joke. :D

You'll want to:

1. start the server and see if the different APIs are working properly.
2. run all of the scripts called by [scripts/update-all](https://github.com/tobie/specref/blob/master/scripts/update-all#L39-L46).

Comment (marcoscaceres) [inline package.json]: Ok, let's see what happens :)

**Comment by marcoscaceres** [comment]
Woah, um... d'oh. Um... I think it just unintentionally deployed from my local branch.

**Comment by marcoscaceres** [comment]
Ah, ok... no. Gave myself a scare... I didn't realize the bot had pushed new changes a few minutes ago.

File: package.json (hunk 2 above)
Comment (marcoscaceres) [inline package.json]: So, output seems ok:

```
> specref@0.2.0 test /private/var/folders/lx/g9xdf_711t9d97bppsxnpwy00000gn/T/specref
> mocha -u tdd -R min --timeout 300000 ./test/*.js

  125037 passing (34s)

Writing output to /private/var/folders/lx/g9xdf_711t9d97bppsxnpwy00000gn/T/specref/test/ref-list.json...
This looks good. About to update.
[2017-01-03T0848-update 689fb54] 2017-01-03T08:50:00 auto update
 2 files changed, 16274 insertions(+), 16258 deletions(-)
To github.com:marcoscaceres/specref.git
   d093639..689fb54  HEAD -> master
```

Comment (marcoscaceres) [inline package.json]: (note, the above also includes the install steps, etc. from running "./scripts/update-all"). No errors.

**Pull request merged**